docs/developer_guide.md

```diff
@@ -64,15 +70,15 @@ In-case of problems, see <a href="#debug">Debugging</a>.
 
 	* With a Docker cache :
 	
-	```bash
```

Comment:
Why we put bash keyword in some places and in some we remove?

Reply:
bash keyword indicate that command will be executed in a test